src/IR/Builder.cpp

```diff
@@ -101,10 +140,10 @@ std::shared_ptr<const FunctionSummary> generateFunctionSummary(const llvm::Funct
           continue;
         }
 
-        auto calledFunc = callInst->getCalledFunction();
-        if (calledFunc == nullptr || !calledFunc->hasName()) {
-          // TODO: Log warning
-          llvm::errs() << "could not find called func: " << *callInst << "\n";
+        auto calledFunc = FunctionSummaryBuilder::resolveTargetFunction(callInst);
+        if (calledFunc == nullptr) {
+          continue;
+        } else if (calledFunc->isIntrinsic() || calledFunc->isDebugInfoForProfiling()) {
           continue;
         }
```

Comment:
Can this be combined into a single condition?

```suggestion
        if (calledFunc == nullptr || calledFunc->isIntrinsic() || calledFunc->isDebugInfoForProfiling()) {
          continue;
        }
```

Reply:
yes, just want to make sure if it is ok if we ignore when 
```calledFunc->isIntrinsic() || calledFunc->isDebugInfoForProfiling()```